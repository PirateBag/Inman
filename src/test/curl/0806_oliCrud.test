
===JSON===

{ "rows" : [
        {  "activityState" : "INSERT",  "itemId" : "3", "parentOliId" : "0", "quantityOrdered" : "500", "quantityAssigned" : "0.0",
        "completeDate" : "2025-0915",  "orderState" : "OPEN",
        "orderType" : "MOHEAD" },

        {  "activityState" : "INSERT",  "itemId" : "3", "parentOliId" : "0", "quantityOrdered" : "1000", "quantityAssigned" : "0.0",
        "completeDate" : "2025-1015", "orderState" : "OPEN",
        "orderType" : "MOHEAD" },

        {  "activityState" : "INSERT",  "itemId" : "3", "parentOliId" : "0", "quantityOrdered" : "1500", "quantityAssigned" : "0.0",
        "completeDate" : "2025-1115",  "orderState" : "OPEN",
        "orderType" : "MOHEAD" },

        {  "activityState" : "INSERT",  "itemId" : "3", "parentOliId" : "0", "quantityOrdered" : "50", "quantityAssigned" : "0.0",
        "completeDate" : "2025-1215",  "orderState" : "OPEN",
        "orderType" : "MOHEAD" }
] }
