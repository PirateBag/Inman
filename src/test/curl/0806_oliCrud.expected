{
  "responseType" : "MULTILINE",
  "errors" : [ {
    "key" : 1,
    "message" : "Added 5 line items to order 15"
  }, {
    "key" : 1,
    "message" : "Added 5 line items to order 20"
  }, {
    "key" : 1,
    "message" : "Added 5 line items to order 25"
  }, {
    "key" : 1,
    "message" : "Added 5 line items to order 30"
  } ],
  "data" : [ {
    "id" : 15,
    "crudAction" : "INSERT",
    "itemId" : 3,
    "quantityOrdered" : 500.0,
    "quantityAssigned" : 0.0,
    "startDate" : "2025-0914",
    "completeDate" : "2025-0915",
    "parentOliId" : 0,
    "orderState" : "OPEN",
    "orderType" : "MOHEAD"
  }, {
    "id" : 20,
    "crudAction" : "INSERT",
    "itemId" : 3,
    "quantityOrdered" : 1000.0,
    "quantityAssigned" : 0.0,
    "startDate" : "2025-1014",
    "completeDate" : "2025-1015",
    "parentOliId" : 0,
    "orderState" : "OPEN",
    "orderType" : "MOHEAD"
  }, {
    "id" : 25,
    "crudAction" : "INSERT",
    "itemId" : 3,
    "quantityOrdered" : 1500.0,
    "quantityAssigned" : 0.0,
    "startDate" : "2025-1114",
    "completeDate" : "2025-1115",
    "parentOliId" : 0,
    "orderState" : "OPEN",
    "orderType" : "MOHEAD"
  }, {
    "id" : 30,
    "crudAction" : "INSERT",
    "itemId" : 3,
    "quantityOrdered" : 50.0,
    "quantityAssigned" : 0.0,
    "startDate" : "2025-1214",
    "completeDate" : "2025-1215",
    "parentOliId" : 0,
    "orderState" : "OPEN",
    "orderType" : "MOHEAD"
  } ]
}